docs/design/jet/024-non-corruptible-snapshots.md

```diff
@@ -43,15 +43,15 @@ which from the point of view of snapshot data consistency can be summarized as f
 9. Schedule next snapshot.
 
 Snapshotting uses alternating pair of maps `__jet.snapshot.<jobId>.<0 or 1>`:
-one contains the last successful snapshot, the other one contains the snapshot currently being performed (if any).
+one contains the last successful snapshot, the other one contains the snapshot currently in progress (if any).
 They are selected using ongoing data map index.
 These maps also contain `SnapshotVerificationRecord`.
 
 Each "chunk" is tagged with `snapshotId` to which it belongs.
 
 Export-only snapshots are handled specially:
 - they only save state
-- they do not commit or roll back any transactions when they are taken (2nd phase is mostly a no-op) 
+- they do not commit or roll back any transactions when they are taken (2nd phase doesn't run) 
```

Comment:
I meant here that 2nd phase operation is sent to all member, but then does nothing special because of the `if (ssContext.isExportOnly() || processor.snapshotCommitFinish(ssContext.isLastPhase1Successful())` condition in the `ProcessorTasklet` state machine. But the coordinator still waits for the responose